### ISSUE TITLE
Use css for initial positioning of the image

### ISSUE THREAD

**Issue by jerguslejko** [kristiankostecky/the-bouncy-castle#2]
Why do you use js to set initial position of the image?

```
$("img").css({top:($(window).height()/2-50), left:($(window).width()/2-50), position: 'absolute'});
````

could be replaced with

```
img {
    width: 100px;
    height: 100px;
    position: absolute;
    left: calc(50%-50px);
    top: calc(50%-50px);
}
```

but you would also need to set `html` and `body` tags to max height:

```
html, body {
    height: 100%;
}
```

not tested though...

**Comment by kristiankostecky** [comment]
Resolved in e225643

**Comment by jerguslejko** [comment]
much better!
